Ideal geometry2015 muon

pietverwilligen opened the pull request:
This Pull Request creates two geometries:
- Ideal2015Muon
- Extended2015Muon
  both contain all the Phase-2 Muon upgrades: ME0, GE1/1, GE2/1, RE3/1, RE4/1 on top of 
  the Run-2 geometry, and without ECAL and HCAL.  

This PR allows the integration of the Muon Upgrade code in 75X, because
now there is a geometry in which ME0 is included.

While testing the Geometry I noticed that the ME0 SimHits were not produced, therefore also a modification to the OscarProducer was necessary. ME0 Pseudo-Digis are not yet produced / kept, but that will be for another PR.

Please test and sign.
Thanks
Piet

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
test 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
test Ideal2015Muon geometry

pietverwilligen committed:
moved to original v1

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
test 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
trial to create 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
test 2015 Geometry + Full Phase-2 Muon Upgrade

pietverwilligen committed:
test 2015 Geometry

pietverwilligen committed:
added missing trackerParameters.xml

pietverwilligen committed:
added missing trackerParameters.xml

pietverwilligen committed:
added missing GEM and ME0 Geometry Builders

pietverwilligen committed:
added missing GEM and ME0 Geometry Builders

pietverwilligen committed:
produce also MuonME0Hits

pietverwilligen committed:
test 2015 Geometry + Full Phase-2 Muon Upgrade only GEN-SIM step

pietverwilligen committed:
forgot a comma

pietverwilligen committed:
forgot a comma

cmsbuild commented:
A new Pull Request was created by @pietverwilligen (Piet Verwilligen) for CMSSW_7_5_X.

Ideal geometry2015 muon

It involves the following packages:

Configuration/Geometry
Geometry/CMSCommonData
Geometry/MuonCommonData
RecoLocalMuon/GEMRecHit
SimG4Core/Application


@civanch, @nclopezo, @cvuosalo, @ianna, @mdhildreth, @cmsbuild, @Dr15Jones, @slava77 can you please review it and eventually sign? Thanks.
@ghellwig, @bellan, @Martin-Grunewald, @jhgoh, @makortel this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@nclopezo you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

pietverwilligen commented:
RecoLocalMuon/GEMRecHit is involved because I have my test files there ... apologies Slava. I did not touch any source code there.
greets
Piet

slava77 commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

slava77 commented:
Piet,
Is this a solution to get things to work for muons now (put the Phase-II muon geometry on top of otherwise Run-2/2015 detector)?

pietverwilligen commented:
Yes, it is exactly that. I had to remove the calorimeters though.
On 7 May 2015 20:29, "Slava Krutelyov" <notifications@github.com> wrote:

> Piet,
> Is this a solution to get things to work for muons now (put the Phase-II
> muon geometry on top of otherwise Run-2/2015 detector)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8995#issuecomment-99969242>.
>

cmsbuild commented:
+1
Tested at: 5786cc01fd18f1a409b77e007edaaeff1c151ded
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4563/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4563/summary.html

dildick commented:
@pietverwilligen About the ME0 pseudo-digis. Indeed, they are not being produced by default. That is why we have a bunch of functions that (https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimMuon/GEMDigitizer/python/customizeGEMDigi.py) can be used in a cmsDriver command. These functions basically do the same as https://github.com/cms-sw/cmssw/blob/CMSSW_6_2_X_SLHC/SLHCUpgradeSimulations/Configuration/python/me0Customs.py#L24. Such an me0Customs file is not to be included in CMSSW_7_5_X though. Instead we have to customize the process using "era" type functions. However, there is no era for 2019 or 2023 upgrade studies yet.

ianna commented on Geometry/CMSCommonData/python/cmsIdealGeometry2015XML_cfi.py:
```diff
@@ -18,6 +18,7 @@
         'Geometry/CMSCommonData/data/cmsBeam.xml',
         'Geometry/CMSCommonData/data/muonMB.xml',
         'Geometry/CMSCommonData/data/muonMagnet.xml',
+        'Geometry/TrackerCommonData/data/trackerParameters.xml',
```
@pietverwilligen - this is part of a separate PR. I think, it is better to remove this file from here.

makortel commented on SimG4Core/Application/plugins/OscarProducer.cc:
```diff
@@ -102,6 +102,8 @@ OscarProducer::OscarProducer(edm::ParameterSet const & p)
   produces<edm::PSimHitContainer>("MuonRPCHits");
   produces<edm::PSimHitContainer>("MuonGEMHits");
+  produces<edm::PSimHitContainer>("MuonME0Hits");
+
```
Please make the same change to OscarMTProducer too (that is already the default in 75X).

pietverwilligen committed:
applied same change also to OscarMTProducer: added ME0 SimHits

pietverwilligen committed:
brought me0customs back from SLHC releases

pietverwilligen committed:
RPCSimAverageNoiseEff model for RPC Digitization + mixLabel for ME0

pietverwilligen committed:
configured / customized for ME0 digi production

pietverwilligen committed:
brought me0customs back from SLHC releases

pietverwilligen committed:
removed changes to cmsIdealGeometry2015XML_cfi.py from pull request

pietverwilligen committed:
removed empty line

pietverwilligen commented:
Hi All

I have updated the pull request, can you please test?

@dildick thanks for pointing out, I had no time Thursday to look into this, but now I have included it. this pull requests has everything to test the ME0 chain working up to the production of digis.

@ianna i removed the two files from the pull request

@makortel i apllied the change also in OscarMTProducer.cc

greets
Piet

ianna commented:
+1

cmsbuild commented:
Pull request #8995 was updated. @civanch, @nclopezo, @cvuosalo, @mdhildreth, @cmsbuild, @slava77 can you please check and sign again.

cmsbuild commented:
+1
Tested at: 514a1c1d6be3fe6be310c8e1575650c8acc9f00f
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/19/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/19/summary.html

cmsbuild commented:
-1 
Tested at: 514a1c1d6be3fe6be310c8e1575650c8acc9f00f
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/22/summary.html

pietverwilligen commented:
It seemed to have failed in the first step of the 4.53 2012B Photon test...
Its not really clear to me why, because that is just the das query, right?

https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/22/runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/
On 11 May 2015 19:06, "cmsbuild" <notifications@github.com> wrote:

> -1
> Tested at: 514a1c1
> <https://github.com/cms-sw/cmssw/commit/514a1c1d6be3fe6be310c8e1575650c8acc9f00f>
> you can see the results of the tests here:
>
> https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/22/summary.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8995#issuecomment-100980314>.
>

slava77 commented:
@cmsbuild what happened here with the tests triggering?
there is a +1 from jenkins at https://github.com/cms-sw/cmssw/pull/8995#issuecomment-100939983 for the same commit and then shortly after another test was done with a -1 https://github.com/cms-sw/cmssw/pull/8995#issuecomment-100980314

nclopezo commented:
+1
Hi, 
They both were done with the same IB, the second round of tests had a DAS issue. We have had some issues with the jenkins server, and after the server is restarted we restart some tests that could have not finished. This is why there are 2 tests one after another. I wrote the "+1" to mark this PR as `tests-approved`.

civanch commented:
+1

slava77 commented:
+1

for https://github.com/cms-sw/cmssw/pull/8995	514a1c1
- reco parts are minimally affected (only configs in RecoLocalMuon/GEMRecHit/test)
- jenkins tests complete ok and show no differences in comparisons, as expected

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_7_5_X IBs unless changes (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @nclopezo, @smuzaffar

davidlange6 commented:
@ianna - is the sim group on board with managing these unphysical geometries in this way? I'm  worried both about the "2015" in the name and wondering if we would be better of putting these in a test directory somewhere instead of the official place to make it more difficult for non-experts to get the wrong answer

civanch commented:
I agree that name/directory for upgrade geometry should be clear. In this PR we do not have a direct clash but a potential  problem in long term. Probably, this PR may be merged but we should re-discuss when we discuss muon upgrade software (likely next week).

pietverwilligen commented:
Hi Matti

When doing some checks I see that the ME0SimHits are produced with
DetUnitId = 0.
Can you point me to the piece of code where they are exactly produced, such
that I can investigate it?


Hi Vladimir, David, I agree that the name of those technical geometries is
a bit unfortunate ...

greets
Piet

On Tue, May 12, 2015 at 3:26 PM, Vladimir Ivantchenko <
notifications@github.com> wrote:

> I agree that name/directory for upgrade geometry should be clear. In this
> PR we do not have a direct clash but a potential problem in long term.
> Probably, this PR may be merged but we should re-discuss when we discuss
> muon upgrade software (likely next week).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8995#issuecomment-101276001>.
>



-- 
Piet Verwilligen

INFN -- Sezione di Bari
Via E. Orabona 4
I-70125 Bari, Italy
Phone: +39 345 74 70 642

makortel commented:
Piet, I don't really know, but my guess would somewhere in
https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSensitiveDetector.h
https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSlaveSD.h

Vladimir could know better.

pietverwilligen commented:
Hi Vladimir

When doing some checks I see that the ME0SimHits are produced with
DetUnitId = 0.
Can you point me to the piece of code where they are exactly produced, such
that I can investigate it?
Thanks
Piet

On Tue, May 19, 2015 at 9:27 AM, Matti Kortelainen <notifications@github.com
> wrote:

> Piet, I don't really know, but my guess would somewhere in
>
> https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSensitiveDetector.h
>
> https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSlaveSD.h
>
> Vladimir could know better.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8995#issuecomment-103379797>.
>



-- 
Piet Verwilligen

INFN -- Sezione di Bari
Via E. Orabona 4
I-70125 Bari, Italy
Phone: +39 345 74 70 642

ianna commented:
@davidlange6 - Extended2015 scenario is the base one for this, hence it is reflected in the name. In any case, the new scenario supposed to be very short-lived and it will not affect the base line. I've already started 2023 line which will replace this new scenario. Something like "Test" could be added to the name, but it has to be in a current place for cmsDriver to pick it up without too much hassle. Non-experts should use DB geometry and I'm sure, one would be making an extra effort to pick this one up.

davidlange6 commented:
@ianna, @pietverwilligen - indeed something like Test, Dev, and maybe GEM too is needed to make it really obvious that this is not a 2015 geometry that could be of interest.

pietverwilligen committed:
Merged refs/pull/8995/head from repository cms-sw

pietverwilligen committed:
update gem11.xml to v7

pietverwilligen committed:
update gem11.xml to v7

pietverwilligen committed:
add GEMDev as tag to load GeometryExtended2015MuonReco_GEMDev_cff.py

pietverwilligen committed:
renamed 2015Muon --> 2015Muon_GEMDev

pietverwilligen committed:
renamed 2015Muon --> 2015Muon_GEMDev

pietverwilligen committed:
renamed 2015Muon --> 2015Muon_GEMDev

pietverwilligen committed:
Merge pull request #8 from cms-sw/CMSSW_7_5_X

PRToFixMess_IdealGeometry2015Muon_PR_v1

pietverwilligen committed:
Delete cmsExtendedGeometry2015MuonXML_cfi.py

remove because of confusing name

pietverwilligen committed:
Delete GeometryExtended2015MuonReco_cff.py

remove because of confusing name

pietverwilligen committed:
Delete GeometryIdeal2015Muon_cff.py

no need for both Ideal2015Muon and Extended2015Muon geometries, since both are test geometries

pietverwilligen committed:
Delete GeometryIdeal2015MuonReco_cff.py

no need for both Ideal2015Muon and Extended2015Muon geometries, since both are test geometries

pietverwilligen committed:
Delete GeometryExtended2015Muon_cff.py

removed because of confusing name

pietverwilligen committed:
Delete cmsIdealGeometry2015MuonXML_cfi.py

no need for both Ideal2015Muon and Extended2015Muon geometries, since both are test geometries

pietverwilligen commented:
I have changed the names to 2015Muon_GEMDev.
I have removed the IdealGeometry2015Muon, since we do not need 2 geometries to do the test ... 
so keeping only GeometryExtended2015Muon that has been renamed to GeometryExtended2015MuonXML_GEMDev

cmsbuild commented:
Pull request #8995 was updated. @civanch, @nclopezo, @cvuosalo, @ianna, @mdhildreth, @cmsbuild, @alja, @Dr15Jones, @slava77 can you please check and sign again.

cvuosalo commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: 6431479f18fafcbc4da2d12f0e2d2704a7d62bd0
 When I ran the RelVals I found an error in the following worklfows: 
 4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

----- Begin Fatal Exception 22-May-2015 22:22:45 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
140.53 step2
<pre>runTheMatrix-results/140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI/step2_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI.log

----- Begin Fatal Exception 22-May-2015 22:23:40 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

----- Begin Fatal Exception 22-May-2015 22:24:25 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD.log

----- Begin Fatal Exception 22-May-2015 22:24:26 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
4.53 step3
<pre>runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step3_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT.log

----- Begin Fatal Exception 22-May-2015 22:24:27 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
1003.0 step2
<pre>runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM/step2_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM.log

----- Begin Fatal Exception 22-May-2015 22:24:30 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
8.0 step3
<pre>runTheMatrix-results/8.0_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS/step3_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS.log

----- Begin Fatal Exception 22-May-2015 22:25:22 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
1306.0 step3
<pre>runTheMatrix-results/1306.0_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15+MINIAODMCUP15/step3_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15+MINIAODMCUP15.log

----- Begin Fatal Exception 22-May-2015 22:28:55 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
1330.0 step3
<pre>runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15+MINIAODMCUP15/step3_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15+MINIAODMCUP15.log

----- Begin Fatal Exception 22-May-2015 22:35:23 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
9.0 step3
<pre>runTheMatrix-results/9.0_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST/step3_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST.log

----- Begin Fatal Exception 22-May-2015 22:37:31 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
25.0 step3
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT/step3_TTbar+TTbar+DIGI+RECO+HARVEST+ALCATT.log

----- Begin Fatal Exception 22-May-2015 22:40:28 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
50202.0 step3
<pre>runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50+MINIAODMCUP1550/step3_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50+MINIAODMCUP1550.log

----- Begin Fatal Exception 22-May-2015 22:48:43 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>
25202.0 step3
<pre>runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+MINIAODMCUP15/step3_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+MINIAODMCUP15.log

----- Begin Fatal Exception 22-May-2015 22:54:59 CEST-----------------------

An exception of category 'PluginLibraryLoadError' occurred while

   [0] Constructing the EventProcessor

Exception Message:

unable to load /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so because /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-05-22-1100/lib/slc6_amd64_gcc491/poisoned/plugin-poisoned-DQMTrackingMonitor.so: cannot open shared object file: No such file or directory

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4966/summary.html 

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/ccc24a7faa35a1dc9d2a4658892c217945dc7d48
https://github.com/cms-sw/cmssw/commit/a96732c76a28ffb4a6da5c95354e23e4892f29a8
https://github.com/cms-sw/cmssw/commit/03989e0a5814861d2f80bd983b5026a6b8fc083c
https://github.com/cms-sw/cmssw/commit/ab290b29f931e25b24a5d2d960c59ef706e4e9bc
https://github.com/cms-sw/cmssw/commit/2310b10305a4d98de50635666fbd28fb3d4bee2d
https://github.com/cms-sw/cmssw/commit/a089676c01c43c755fd072f7422693847665705d
https://github.com/cms-sw/cmssw/commit/6b3b490c704af537565df7c84e7a11f52692e789
https://github.com/cms-sw/cmssw/commit/4bc517995252dd3623c349964f192c872a237fb5
https://github.com/cms-sw/cmssw/commit/dadd2f8982450c18006dc0df514ebda73f53b04c
https://github.com/cms-sw/cmssw/commit/cde58a0fb3f0c3699b7e2c905cc47c607eaa090b
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4966/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4966/git-merge-result

slava77 commented:
@cmsbuild please test
new IB CMSSW_7_5_X_2015-05-23-1100 is in and seems ok at least on the build side

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 6431479f18fafcbc4da2d12f0e2d2704a7d62bd0
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4973/summary.html

civanch commented:
+1

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/4973/summary.html

slava77 commented:
+1

for https://github.com/cms-sw/cmssw/pull/8995	6431479
- reco is minimally affected (only files in /test)
- jenkins tests completed and show no relevant differences in comparisons with the baseline 
    - IsoTkMu HLT plots in DQM have occasional small differences, which persists in IB tests for the past few days already

alja commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_7_5_X IBs unless changes (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @nclopezo, @smuzaffar

davidlange6 commented:
@pietverwilligen - thanks - I worry that someone is assuming that '_' are not part of the geometry name. Could you just remove the _ and I'll merge the PR

pietverwilligen committed:
renamed

pietverwilligen committed:
renamed

pietverwilligen committed:
renamed

pietverwilligen committed:
Merge branch 'IdealGeometry2015Muon' of https://github.com/pietverwilligen/cmssw into IdealGeometry2015Muon

pietverwilligen committed:
Delete GeometryExtended2015MuonReco_GEMDev_cff.py

pietverwilligen committed:
Delete GeometryExtended2015Muon_GEMDev_cff.py

pietverwilligen committed:
Delete cmsExtendedGeometry2015MuonXML_GEMDev_cfi.py

pietverwilligen committed:
Delete SingleMuPt100_cfi_GEM-SIM-DIGI_Extended2015_cfg.py

pietverwilligen committed:
Delete SingleMuPt100_cfi_GEM-SIM-DIGI_Ideal2015Muon_cfg.py

pietverwilligen committed:
Delete SingleMuPt100_cfi_GEM-SIM-DIGI_Ideal2015_cfg.py

pietverwilligen committed:
rename

pietverwilligen commented:
done, please test again.

cmsbuild commented:
Pull request #8995 was updated. @civanch, @nclopezo, @cvuosalo, @ianna, @mdhildreth, @cmsbuild, @alja, @Dr15Jones, @slava77 can you please check and sign again.

pietverwilligen commented:
Hi Matti

Thanks

I found that simhits are produced exactly in
https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/src/MuonSensitiveDetector.cc

Thanks, very useful
greets
Piet

On Tue, May 19, 2015 at 9:27 AM, Matti Kortelainen <notifications@github.com
> wrote:

> Piet, I don't really know, but my guess would somewhere in
>
> https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSensitiveDetector.h
>
> https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/SimG4CMS/Muon/interface/MuonSlaveSD.h
>
> Vladimir could know better.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8995#issuecomment-103379797>.
>



-- 
Piet Verwilligen

INFN -- Sezione di Bari
Via E. Orabona 4
I-70125 Bari, Italy
Phone: +39 345 74 70 642

ianna commented:
+1

cmsbuild commented:
+1
Tested at: a461e5166dcd7044b0aabd6fd1ce3e0d4282c19c
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/5113/summary.html

civanch commented:
+1

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/5113/summary.html

slava77 commented:
+1

for https://github.com/cms-sw/cmssw/pull/8995	a461e51
- reco is minimally affected (only config files in /test)
- jenkins tests pass and comparisons with the baseline show no difference

cmsbuild commented:
-1 
Tested at: a461e5166dcd7044b0aabd6fd1ce3e0d4282c19c
 The relvals timed out after 2 hours. 
 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8995/7919/summary.html
